### PR TITLE
Devi/external process launcher

### PR DESCRIPTION
**Pull request by v-dmudili**
Created a folder for External Process Launcher app & added all the project files for windows 10 Iot Enterprise.

Following changes :

1) file modified : ProcessLauncherSample.csproj
     Updated the Target Platform Version from 10.0.17763 to 10.0.19041
     Updated the Target Platform MinVersion from 10.0.10240.0 to 10.0.15063

2) file modified : SampleConsoleApplication.vcxproj
     Updated the Target Platform Version from 10.0.17763 to 10.0.19041
     Updated the PlatformToolset from v141 to v142

2) file modified : README.md
    Adding windows 10 iot Enterprise in products.
    Generate and Installing App Package procedure.
    Updated the Windows Platform from IoT Core to IoT Enterprise.

### CODE REVIEW

File: ExternalProcessLauncher/CS/.gitkeep [hunk 1]
```diff
@@ -0,0 +1 @@
+
```
Comment (neilsh-msft) [inline ExternalProcessLauncher/CS/.gitkeep:1]: Unneeded .gitkeep file.  Remove?

File: ExternalProcessLauncher/.gitkeep [hunk 2]
```diff
@@ -0,0 +1 @@
+
```
Comment (neilsh-msft) [inline ExternalProcessLauncher/.gitkeep:1]: Unneeded .gitkeep file. Remove?

File: ExternalProcessLauncher/CS/ProcessLauncherSample/.gitkeep [hunk 3]
```diff
@@ -0,0 +1 @@
+
```
Comment (neilsh-msft) [inline ExternalProcessLauncher/CS/ProcessLauncherSample/.gitkeep:1]: Unneeded .gitkeep file. Remove?

File: ExternalProcessLauncher/CS/ProcessLauncherSample/Assets/.gitkeep [hunk 4]
```diff
@@ -0,0 +1 @@
+
```
Comment (neilsh-msft) [inline ExternalProcessLauncher/CS/ProcessLauncherSample/Assets/.gitkeep:1]: Unneeded .gitkeep file. Remove?

File: ExternalProcessLauncher/CS/ProcessLauncherSample/Properties/.gitkeep [hunk 5]
```diff
@@ -0,0 +1 @@
+
```
Comment (neilsh-msft) [inline ExternalProcessLauncher/CS/ProcessLauncherSample/Properties/.gitkeep:1]: Unneeded .gitkeep file. Remove?

File: ExternalProcessLauncher/CS/SampleConsoleApplication/.gitkeep [hunk 6]
```diff
@@ -0,0 +1 @@
+
```
Comment (neilsh-msft) [inline ExternalProcessLauncher/CS/SampleConsoleApplication/.gitkeep:1]: Unneeded .gitkeep file. Remove?

File: Resources/.gitkeep [hunk 7]
```diff
@@ -0,0 +1 @@
+
```
Comment (neilsh-msft) [inline Resources/.gitkeep]: Unneeded .gitkeep file. Remove?

File: Resources/images/ProcessLauncherSample/.gitkeep [hunk 8]
```diff
@@ -0,0 +1 @@
+
```
Comment (neilsh-msft) [inline Resources/images/ProcessLauncherSample/.gitkeep]: Unneeded .gitkeep file. Remove?

File: Resources/images/.gitkeep [hunk 9]
```diff
@@ -0,0 +1 @@
+
```
Comment (neilsh-msft) [inline Resources/images/.gitkeep]: Unneeded .gitkeep file. Remove?

File: ExternalProcessLauncher/CS/ProcessLauncherSample/ProcessLauncherSample.csproj [hunk 10]
```diff
@@ -0,0 +1,156 @@
...
+      <SubType>Designer</SubType>
+    </Page>
+  </ItemGroup>
+  <ItemGroup>
+    <PackageReference Include="Microsoft.NETCore.UniversalWindowsPlatform">
+      <Version>5.0.0</Version>
+    </PackageReference>
+  </ItemGroup>
+  <ItemGroup>
+    <SDKReference Include="WindowsDesktop, Version=10.0.19041.0">
```
Comment (neilsh-msft) [inline ExternalProcessLauncher/CS/ProcessLauncherSample/ProcessLauncherSample.csproj:141]: Was this SDKReference addition necessary?  Does VS 2019 require it?

File: ExternalProcessLauncher/CS/ProcessLauncherSample/MainPage.xaml.cs [hunk 11]
```diff
@@ -0,0 +1,97 @@
...
+        }
+
+        private async Task RunProcess()
+        {
+            var options = new ProcessLauncherOptions();
+            var standardOutput = new InMemoryRandomAccessStream();
+            var standardError = new InMemoryRandomAccessStream();
+            options.StandardOutput = standardOutput;
+            options.StandardError = standardError;
+
```
Comment (neilsh-msft) [inline ExternalProcessLauncher/CS/ProcessLauncherSample/MainPage.xaml.cs:40]: Extra blank line added from original.  Please remove.

**Review by rsameser — APPROVED** [review]

File: ExternalProcessLauncher/CS/.gitkeep (hunk 1 above)
Comment (v-abdrahman) [reply ExternalProcessLauncher/CS/.gitkeep:1]: Removed

File: ExternalProcessLauncher/.gitkeep (hunk 2 above)
Comment (v-abdrahman) [reply ExternalProcessLauncher/.gitkeep:1]: Removed

File: ExternalProcessLauncher/CS/ProcessLauncherSample/.gitkeep (hunk 3 above)
Comment (v-abdrahman) [reply ExternalProcessLauncher/CS/ProcessLauncherSample/.gitkeep:1]: Removed

File: ExternalProcessLauncher/CS/ProcessLauncherSample/Assets/.gitkeep (hunk 4 above)
Comment (v-abdrahman) [reply ExternalProcessLauncher/CS/ProcessLauncherSample/Assets/.gitkeep:1]: Removed

File: ExternalProcessLauncher/CS/ProcessLauncherSample/Properties/.gitkeep (hunk 5 above)
Comment (v-abdrahman) [reply ExternalProcessLauncher/CS/ProcessLauncherSample/Properties/.gitkeep:1]: Removed

File: ExternalProcessLauncher/CS/SampleConsoleApplication/.gitkeep (hunk 6 above)
Comment (v-abdrahman) [reply ExternalProcessLauncher/CS/SampleConsoleApplication/.gitkeep:1]: Removed

File: Resources/.gitkeep (hunk 7 above)
Comment (v-abdrahman) [reply Resources/.gitkeep]: Removed

File: Resources/images/ProcessLauncherSample/.gitkeep (hunk 8 above)
Comment (v-abdrahman) [reply Resources/images/ProcessLauncherSample/.gitkeep]: Removed

File: Resources/images/.gitkeep (hunk 9 above)
Comment (v-abdrahman) [reply Resources/images/.gitkeep]: Removed

File: ExternalProcessLauncher/CS/ProcessLauncherSample/MainPage.xaml.cs (hunk 11 above)
Comment (v-abdrahman) [reply ExternalProcessLauncher/CS/ProcessLauncherSample/MainPage.xaml.cs:40]: Removed the blank line.

File: ExternalProcessLauncher/CS/ProcessLauncherSample/ProcessLauncherSample.csproj (hunk 10 above)
Comment (v-abdrahman) [reply ExternalProcessLauncher/CS/ProcessLauncherSample/ProcessLauncherSample.csproj:141]: Yes, It is necessary. It got updated while setting up the sample in VS2019.

**Pull request merged**